data/contract_docs/contract.md

```diff
@@ -81,6 +82,25 @@ However, it needs to be possible to add support for this case. For now, the foll
 * The borders between the types are derived from adjacent frames (within the array of frames) that share the same data type.
 * If a reader does not opt-in into multi-type responses, it should be able to get the first data type that matches what the reader is looking for.
 
+## Versioning
+
+An object of this contract is to be as stable as possible. When `1.0` is reached, changes should be limited to enhancements until a `2.0` version, which is something that should generally be avoided in datatype versions.
```

Comment:
Roughly copying what @sdboyer is doing with https://github.com/grafana/thema  . Partly in case we ever get to a point where we can (and it makes sense) to use those schema definitions for this. Also, since each datatype is getting its own version, a two point version seems simpler.

Reply:
me...and other data people! 😉 

https://docs.open-metadata.org/connectors/ingestion/versioning

Reply:
With the `[]number` array approach -- what is the pattern for tagging things as beta/dev/pre-release?